deployment/ccip/test.helpers.go

```diff
@@ -0,0 +1,34 @@
+package ccipdeployment
+
+import (
+	"math/big"
+)
+
+const (
```

Comment:
These consts and vars are some of the few things kept in deployment, as the rest is for tests, and only integration tests, and *_test.go files use them.